rootfs/etc/nginx/lua/monitor.lua

```diff
@@ -4,13 +4,13 @@ local assert = assert
 local new_tab = require "table.new"
 local clear_tab = require "table.clear"
 local clone_tab = require "table.clone"
-local nkeys = require "table.nkeys"
```

Comment:
My understanding was that this is performant. Do you have any more detail how bad is this in comparision to counting explicitly?

Reply:
Please refer to: https://github.com/openresty/luajit2/blob/v2.1-agentzh/src/lj_tab.c#L721
It traverses the entire array and hash parts, and skip the nil item.